### PR TITLE
feat(Improve structure and content of NRQL math page)

### PR DESCRIPTION
**Pull request by dbarnesbrownNR**
Reordered existing content and did a big clarity edit. Note that this pull request does not include a new examples and sample workflows section, because that work will be dependent on the availability of a NRQL SME to partner with the tech writer doing the update.

### CODE REVIEW

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx [hunk 1]
```diff
@@ -38,45 +38,56 @@ SELECT count(*)/uniqueCount(session) FROM PageView
 SELECT average(duration-databaseDuration) FROM Transaction
 ```
 
-## Results with STRING or FLOAT [#strings]
+## Use advanced math operators with SELECT [#advancedmath]
 
-Here is how NRQL handles strings present in math calculations:
+NRQL also includes some advanced mathematical capabilities that can be used for complex calculations. This is helpful if you want to process data to display more effectively in the UI, or make statistical or psychometric calculations on queried results in a single step.
```
Comment (paperclypse) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx:43]: "...display it more effectively..."?

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx [hunk 2]
```diff
@@ -38,45 +38,56 @@ SELECT count(*)/uniqueCount(session) FROM PageView
...
-This will replace NULL values with 0 or any number specified.
+Similarly `clamp_min(duration, 1)` will not return any value lower than 1. 
 
-<Callout variant="tip">
-  This can also be used to test whether something returns NULL or zero. `(zero) OR 1` returns 0. `(NULL) OR 1` returns 1.
-</Callout>
+The following chart shows the result of clamping both min and max to keep the value in the range 70-90.
 
-## Advanced math functions
+![clamp.png](./images/clamp.png "clamp.png")
```
Comment (paperclypse) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx:59]: It would be good to have some alt text in there describing the image.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx [hunk 3]
```diff
@@ -38,45 +38,56 @@ SELECT count(*)/uniqueCount(session) FROM PageView
...
+
+These functions compute the logarithm of the argument for various bases.
+
+* `ln(n)` computes the natural logarithm: the logarithm base e.
+* `log2(n)` computes the logarithm base 2.
+* `log10(n)` computes the logarithm base 10.
+* `log(n, b)` allows logarithms to be computed with an arbitrary base b.
+* All logarithms satisfy the identity: `log(pow(b, n), b) = n.`
+
+Note that `log(0`) is undefined, for all bases. Be aware that if you take the logarithm of something that might be zero, you may end up getting "No Value" back from your query.
```
Comment (paperclypse) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx:79]: Maybe put "No Value" in backticks `No Value`?

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx [hunk 4]
```diff
@@ -38,45 +38,56 @@ SELECT count(*)/uniqueCount(session) FROM PageView
...
+### Rounding functions: `round`, `floor`, `ceil` [#rounding]
 
 These three functions force decimal numbers to one of the neighboring integers.
 
 * `floor(n)` returns the closest integer less than or equal to _n_.
-* `ceil(n)` returns the closest integer greater than or equal to _n_.
+* `ceil(n)` (short for "ceiling") returns the closest integer greater than or equal to _n_.
 * `round(n)` returns the closest integer to _n_ in either direction.
 
 ![floor-round-ceil.png](./images/floor-round-ceil.png "floor-round-ceil.png")
```
Comment (paperclypse) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx:93]: Some alt text would be good for this image too.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx [hunk 5]
```diff
@@ -85,40 +96,33 @@ These three functions force decimal numbers to one of the neighboring integers.
...
-* `log10(n)` computes the logarithm base 10.
-* `log(n, b)` allows logarithms to be computed with an arbitrary base b.
-* All logarithms satisfy the identity: `log(pow(b, n), b) = n.`
+<Callout variant="tip">
+  This can also be used to test whether something returns NULL or zero. `(zero) OR 1` returns 0. `(NULL) OR 1` returns 1.
+</Callout>
 
-Note that `log(0`) is undefined, for all bases. Be aware that if you take the logarithm of something that might be zero, you may end up getting `No Value` back from your query.
\ No newline at end of file
+## For more help [#more-help]
```
Comment (paperclypse) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx:128]: I don't think you need to add "## For more help". Pretty sure it's added automatically. Right now, it's just showing the header without anything beneath it.

**Review by paperclypse — APPROVED** [review]
A pleasure to read, which I can't say about too many math-related things I've read :D Approved with some minor comments.
